packages/components/src/Calendar/Calendar.module.scss

```diff
@@ -0,0 +1,214 @@
+@import "~@kaizen/design-tokens/sass/typography";
```

Comment:
What imports this `Calendar.module.scss` file?

Reply:
`calendarClassNames.ts` 🫠 Terrible isn't it haha. It's base styles that apply to both types of Calendars.

Reply:
Should I rename this to `CalendarBase.module.scss` instead? Would that be clearer?

Reply:
Probably align it to what imports it lol